docs/documentation/e2e.md

```diff
@@ -15,6 +15,16 @@ End-to-end tests are run via [Protractor] (https://angular.github.io/protractor/
 
 ## Options
 <details>
+  <summary>base href</summary>
```

Comment:
Actually what happens is that all flags supported by `ng serve` are also supported by `ng e2e`. I thought this was mentioned here but it's not. 

Instead of adding the single flag, could you mention that please?

Reply:
Oh ok, that kinda makes sense. I've been using it in conjunction with `--no-serve` which is why I thought it was actually an `ng e2e` command.

I'll change it to say what you suggest though.